From 305a48f529c24b1e8c64bb4116b1e212d02995a2 Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Thu, 5 Jan 2023 16:44:18 +0800
Subject: [PATCH] Revert "Linux 5.10.162"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 0fe4548663f7dc2c3b549ef54ce9bb6d40a235be.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 33422c7d149e..14bb1bb37770 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 162
+SUBLEVEL = 161
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.3

